queries/contributions.sql

```diff
@@ -9,11 +9,9 @@ WITH RelevantContributions AS (
     m.id as mediaitem_id
   FROM
     public.mediaitems m
-    INNER JOIN public.mediaitems_contributions mc ON m.id = mc.mediaitems_id
-    INNER JOIN contributions c ON mc.contributions_id = c.id
-    AND c.person_id = ANY (@person_ids::uuid[])
-  where
-    m.primary_episode_id is not null
+  INNER JOIN contributions c ON c.mediaitem_id = m.primary_episode_id
+    and c.person_id = ANY (@person_ids::uuid[])
+    and m.primary_episode_id is not null
```

Comment:
Just curious, is there a difference in the query plan/preformance if you have this in `ON` or if you split it into a `WHERE`